code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -281,6 +281,15 @@
 	list_reagents = list(/datum/reagent/consumable/honey = 10, /datum/reagent/consumable/nutriment = 0.5, /datum/reagent/consumable/sugar = 2)
 	bitesize = 2
 
+/obj/item/reagent_containers/food/snacks/pillpacket
+	name = "crumpled pill packet"
+	desc = "Fully biodegradable, some say it's delicious."
+	icon = 'icons/obj/items/trash.dmi'
+	icon_state = "pillpacketempty"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 1) //Done just so that it makes the munching sound
+	tastes = list("syrupy medicine" = 1)
+	bitesize = 0.1
```

Comment:
Can we not have ungas eating plastic wrappers please

Reply:
Fine